### ISSUE TITLE
buffering and sometimes channel doesnt play at all when fast forwarding or rewinding in timeshift

### ISSUE THREAD

**Issue by liam8888999** [kodi-pvr/pvr.hts#544]
when rewound the channel begins buuffering after fastforwarding and often gets to 100% but doesnt play.
tvh server and kodi on the same machine so its not a network problem.
when fastforwarded back to live it works correctly

**Comment by liam8888999** [comment]
i use the pass profile on tvheadend and pvr.hts if this makes a difference

**Comment by agitate** [comment]
@liam8888999 I used to have a similar thing happen, about 1 in a 100 times, but that was when fast forwarding and rewinding recorded programs. The blue buffering icon would appear with 100 in the centre, but playback wouldn't start. Stopping and starting the playback would not work. Only rebooting my Nvidia Shield would get it working again. I then increased the playback buffer to 128 in the pvr configuration (Streaming settings -> Stream read chunk size for recordings), and that seemed to make it happen much less, maybe 1 in a 1000 times, so I'd learnt to live with it having only the occasional freeze. However, I've just installed version 19.3 of Kodi which comes with 19.0.0 of the hts add-on and the problem is much much worse. It is happening probably 1 in 4 times now (just watched a 1 hour recording and it froze whilst fast forwarding through 2 of the advert breaks). I've now upped the playback buffer to 256 and that seems to have helped again, having fast forwarded and rewound about 60 times now and not one freeze. Early days to say that that's fixed it, but something to try whilst waiting for a more insightful response from the devs :+1:.
